### ISSUE TITLE
[2.0] Dev Screens Opt In

### ISSUE THREAD

**Issue by GantMan** [infinitered/ignite#557]
Some people don't want the dev screens.

Here's my proposed behavior:

### Add
```sh
$ ignite add dev-screens
Do you want Ignite dev screens to assist you in the app building process? (Y/n)
```

**If Yes:**
* Set Examples to "classic" in Ignite config
* Copy the the screens to containers folder
* Set App/Config/DebugSettings.js `showDevScreens` to `__DEV__`
* Insert a function that renders the dev screens as part of the JSX in the navigation
* Call the function in the navigation, which adds/provides the dev screens
* Add a dev screens button tastefully to the launch page

**If No:** 
_nothing_

### Remove
```
$ ignite remove dev-screens
Are you sure?  This will remove all existing and future examples from being part of this project (Y/n)
```

**If Yes:**
* Set Examples to "false" in Ignite config
* Delete screens from containers folder
* Remove App/Config/DebugSettings.js `showDevScreens` value
* Remove function and call from navigation
* Remove dev screens button from launch page, if launch page is found

**If No:**
_nothing_

**Comment by kevinvangelder** [comment]
The interactive command should also list out what screens will be generated. Something like: `(Example Components, Theme, ...)`

**Comment by kevinvangelder** [comment]
Closed by #590. We will need to hook in to the navigation functionality, but everything else is there.
